File: src/main/java/me/testaccount666/serversystem/userdata/money/vault/EconomyVaultAPI.java
package me.testaccount666.serversystem.userdata.money.vault;

import me.testaccount666.serversystem.ServerSystem;
import org.bukkit.Bukkit;
import org.bukkit.plugin.ServicePriority;

public class EconomyVaultAPI {

    public static boolean isVaultInstalled() {
        return Bukkit.getServer().getPluginManager().getPlugin("Vault") != null;
    }

    public static void initialize() {
        Bukkit.getServer().getServicesManager().register(net.milkbowl.vault.economy.Economy.class, new VaultEconomyProvider(), ServerSystem.Instance, ServicePriority.High);
    }
}

File: src/main/kotlin/me/testaccount666/serversystem/userdata/money/vault/EconomyVaultAPI.kt
package me.testaccount666.serversystem.userdata.money.vault

import me.testaccount666.serversystem.ServerSystem
import net.milkbowl.vault.economy.Economy
import org.bukkit.Bukkit
import org.bukkit.plugin.ServicePriority

object EconomyVaultAPI {
    val isVaultInstalled: Boolean
        get() = Bukkit.getServer().pluginManager.getPlugin("Vault") != null

    fun initialize() {
        Bukkit.getServer().servicesManager.register(
            Economy::class.java, VaultEconomyProvider(),
            ServerSystem.instance, ServicePriority.High
        )
    }
}